crystalfontz/client.py

```diff
@@ -123,18 +124,44 @@
 R = TypeVar("R", bound=Response)
 Result = Tuple[Exception, None] | Tuple[None, R]
 ReportHandlerMethod = Callable[[R], Coroutine[None, None, None]]
+V = TypeVar(name="V")
+
+
+def retry(
```

Comment:
This decorator is only used for except. But it's in a decorator format so I can more easily test it - recall that testing except is difficult, because I can't instrument the queue beforehand.